src/components/proofEditor/proofEditor.module.scss

```diff
@@ -0,0 +1,15 @@
+.proof-editor {
+	
+}
+
+.textInput {
+	display: block;
+	width: 90%;
+	margin-top: 10;

```

Comment:
What unit is this? Can you use `em` or `rem` instead?

Reply:
Since you're setting three margin components here, you could also do `margin: 10 auto 0 auto` instead.